packages/ui5-application-writer/templates/optional/typescript/package.json

```diff
@@ -8,7 +9,8 @@
         "@sapui5/ts-types-esm": "<%- ui5.typesVersion %>",
         "ui5-tooling-transpile": "^0.3.7",
         "typescript": "^4.6.3",
-        "@typescript-eslint/eslint-plugin": "^5.17.0",
-        "@typescript-eslint/parser": "^5.17.0"
+        "@typescript-eslint/eslint-plugin": "^5.59.0",
+        "@typescript-eslint/parser": "^5.59.0",
+        "@sap-ux/eslint-plugin-fiori-tools": "^0.1.0"
```

Comment:
Not sure if there is an easy way to set the version based on `"workspace:*"` like the module package.json's with pnpm